spec/index.html

```diff
@@ -8942,15 +8942,15 @@ <h3>Basic Graph Patterns</h3>
           <a href="https://en.wikipedia.org/w/index.php?title=Multiset">multiset</a>, 
           also known as a <i>bag</i>. A multiset is an
           unordered collection of elements in which each element may appear more than once. It is
-          described by a set of elements and a cardinality function giving the number of occurrences of
-          each element from the set in the multiset.</p>
+          described by a set of elements and a function giving the multiplicity of each of these
+          elements (i.e., the number of times the element is contained in the multiset).</p>
         <p>Write μ for solution mappings.</p>
         <p>Write μ<sub>0</sub> for the mapping such that dom(μ<sub>0</sub>) is the empty set.</p>
         <p>Write Ω<sub>0</sub> for the multiset consisting of exactly the empty mapping
-          μ<sub>0,</sub> with cardinality 1. This is the join identity.</p>
+          μ<sub>0,</sub> with multiplicity 1. This is the join identity.</p>
         <p>Write μ(x) for the solution mapping variable x to RDF term t : { (x, t) }</p>
         <p>Write Ω(x) for the multiset consisting of exactly μ(?x-&gt;t), that is, { { (x, t) } }
```

Comment:
```suggestion
          <var>μ<sub><var>0</var></sub></var>, with multiplicity 1. This is the join identity.</p>
        <p>Write <var>μ(<var>x</var>)</var> for the solution mapping variable <var>x</var> to RDF term <var>t</var> : { (<var>x</var>, <var>t</var>) }.</p>
        <p>Write <var>Ω(<var>x</var>)</var> for the multiset consisting of exactly <var>μ(<var>?x</var>-&gt;<var>t</var>)</var>, that is, <code>{ { (x, t) } }</code>
```

Reply:
Similar issues for this change suggestion as for the other one. That is, `<var>...</var>` should not be used to mark up whole formulas but to mark up the individual symbols within the formulas.

Reply:
Presumably `<var>` shouldn't nest.

Reply:
> [@afs] Presumably `<var>` shouldn't nest.

I think there are cases where it should nest. For instance, consider a sequence of solution mappings and we want to write a definition that iterates over these sequence. Then, we may want to use a symbol (say, _i_) to denote the current iteration step and a symbol to denote the _i_-th solution mapping. That latter symbol may contain the symbol _i_ as a part of it. In this case, I would write `<var>μ<sub><var>i</var></sub></var>`.

Reply:
@hartig @afs — Understanding more now, I think the example `<var>μ<sub><var>i</var></sub></var>` should rather be `<var>μ</var><sub><var>i</var></sub>`, i.e., no nested `<var>`. How these render may change my opinion of which should be used.

I have adjusted my suggested changes, above.

Reply:
> [@TallTed] @hartig @afs — Understanding more now, I think the example `<var>μ<sub><var>i</var></sub></var>` should rather be `<var>μ</var><sub><var>i</var></sub>`, i.e., no nested `<var>`. [...]

No, the nesting is correct in this case. There are two symbols here. For one, there is the symbol that denotes the iteration counter, which is _i_. Hence, we would write `<var>i</var>` for that one. Then, there is the symbol that denotes the _i_-th solution mapping, which is not just _μ_ but _μ_<sub>_i_</sub>. Consequently, the `<var>` tag for this symbol needs to be wrapped around the whole thing (i.e., around all of _μ_<sub>_i_</sub>). Yet, since some part of this second symbol happens to be the first symbol (i.e., _i_), this first symbols also needs to be marked up inside the markup of the second symbol. That's how the HTML/Respec representation of the second symbol becomes nested; `<var>μ<sub><var>i</var></sub></var>`.

Regarding your adjusted change suggestion, the first change in it is still incorrect. Instead of `<var>μ</var><sub><var>0</var></sub>`, it should be `<var>μ<sub>0</sub></var>`, because i) `0` is a constant and not the name of a symbol used in the formula and ii) the symbol in the formula is not just _μ_ but _μ_<sub>0</sub>.

Reply:
Thank you, @hartig.

I understand and have applied `<var>μ<sub>0</sub></var>` in place of `<var>μ</var><sub><var>0</var></sub>` above.

And I think I understand the reason for `<var>μ<sub><var>i</var></sub></var>` instead of my previous thinking of `<var>μ</var><sub><var>i</var></sub>`.

(I do note that `<var>μ<sub>0</sub></var>` and `<var>μ</var><sub><var>0</var></sub>` visually render the same — <var>μ<sub>0</sub></var> and <var>μ</var><sub><var>0</var></sub>, respectively — as do `<var>μ</var><sub><var>i</var></sub>` and `<var>μ<sub>i</sub></var>` — <var>μ</var><sub><var>i</var></sub> and <var>μ<sub>i</sub></var>, respectively — so this pursuit of "correct" markup is a somewhat academic exercise, albeit one which I believe is worth performing.)

Reply:
@TallTed 

> (I do note that `<var>μ<sub>0</sub></var>` and `<var>μ</var><sub><var>0</var></sub>` visually render the same — <var>μ<sub>0</sub></var> and <var>μ</var><sub><var>0</var></sub>, respectively — as do `<var>μ</var><sub><var>i</var></sub>` and `<var>μ<sub>i</sub></var>` — <var>μ</var><sub><var>i</var></sub> and <var>μ<sub>i</sub></var>, respectively — so this pursuit of "correct" markup is a somewhat academic exercise, albeit one which I believe is worth performing.)

Yes, they render the same. However, in the case of `<var>μ</var><sub><var>i</var></sub>`, if you click on any other occurrence of _i_, the _i_ inside <var>μ</var><sub><var>i</var></sub> gets highlighted, which makes it easy to spot all the places where the symbol _i_ is used. In contrast, when using only `<var>μ<sub>i</sub></var>`, the symbol _i_ inside would not get highlighted. (Note that this 'clicking on symbols' does not work in the previews of PRs.)

Thanks for updating your change suggestion. I will commit it.

Reply:
Note that if you add `class="algorithm"` someplace in the path containing the `<var>`, ReSpec provides individual highlighting based on each distinct variable. So, the complete variable should be surrounded by `<var>..</var>`. In this case, that argues for `<var>μ<sub>0</sub></var>`, as `μ<sub>0</sub>` is the variable. Otherwise, it would treat `μ` and `<sub>0</sub>` as separate variables. You can actually put the algorithm class on the body if all equivalent variable names are considered the same. Sadly, this seems to be lost when publishing to /TR, although I've seen things lately that could survive publication.

Reply:
That's what I meant by 'clicking on symbols'. It is a really nice feature. It would be a shame if this highlightning disappears when publishing.

Reply:
There's been a ReSpec issue open on this since 2021: https://github.com/w3c/respec/issues/3765. I believe the issue is that it uses JavaScript to make it work, and this was discouraged in published documents. I'm sure it could be done with just CSS, though.